### PR TITLE
Added april 2019 meeting notes

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

File: meetings/2019-04-09.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# TSC Meeting April 9, 2019 @12PM MST
+
+## [Hangouts Link](https://hangouts.google.com/hangouts/_/doi.gov/jlaura)
+
+## Agenda
+- Organizing a planetary science software group.
```
Comment (SgStapleton) [inline meetings/2019-04-09.md]: Was there nothing else...?

Comment (jessemapel) [reply meetings/2019-04-09.md]: I'm waiting for @jlaura to post last meeting's notes with stuff to continue discussion of. Also, people can add other agenda items. This is just something that we talked about here at LPSC.

Comment (jlaura) [reply meetings/2019-04-09.md]: My slacker self got the PR opened.

**Comment by jessemapel** [comment]
@all is this good to be merged so that we can start adding agenda items?

**Pull request merged**